README.rdoc

```diff
@@ -1,5 +1,9 @@
 = Foreman API bindings for Ruby
 
+== NOTE
+
+Alternatively, you may want to take a look at {apiepie}[https://github.com/Apipie/apipie-bindings]. It is the replacement for foreman_api that is used in the Foreman CLI. It has more frequent releases. Additional info at {rubydoc}[http://rubydoc.info/github/Apipie/apipie-bindings/frames/file/README.md].
```

Comment:
Thanks for the patch! It would be better to name the link apipie-bindings to avoid confision with apipie library itself.